meta/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+dependencies: []
+
+galaxy_info:
+  author: Fritz Otlinghaus
+  description: Kibana instance.
+  company: "stuvus"
+  license: "license (BSD, MIT)"
```

Comment:
I thought it was BY-NC?
Please populate this file exactly like in our skeleton.